server/templates/partials/PrinterManager/healthChecks.modal.ejs

```diff
@@ -12,40 +12,37 @@
         <div class="col-lg-12">
             <div class="collapse" id="collapseExample">
                 <div class="alert alert-secondary" role="alert" id="healthChecksHelpText">
-                    <p>OctoFarm will periodically run a few checks on your printers just to make sure nothing is out of
-                        whack! Each check will either pass (Green icon) or fail (Red icon). Clicking a red icon will
-                        give you an information box on the problem, OctoFarms reaction and finally what solution you can
+                    <p>OctoFarm will periodically run checks on your printers to make sure nothing is out of
+                        whack! Each check will either pass (reen icon) or fail (ed icon). Clicking a red icon will
```

Comment:
Should be Green and Red

Reply:
Ah great catch.  I've added a commit to address that.